ci/cron/src/Main.hs

```diff
@@ -463,9 +457,4 @@ main = do
       Docs -> do
           docs sdkDocOpts
           docs damlOnSqlDocOpts
-          -- FIXME: this is meant to run once to create the `latest` file
-          -- without waiting for the next stable release.
-          gh_versions <- fetch_gh_versions (includedVersion sdkDocOpts)
```

Comment:
I've decided against keeping this long-term, because the only case in which this code makes sense is when renaming files, and I'm not doing that again.

Reply:
Don’t we also need this if we add a new file? Not that I have anything in mind that we might want to add but `latest` has been pretty recent so it seems at least feasible we’ll need this again. Of course, we can always add it once and remove it afterwards in that case but the "add it once and remove it again" stuff has been required quite often now and we’ve messed it up occasionally and had to add it in a separate PR. Is there a downside to keeping this apart from the fact that we push it even if nothing changed (which should be cheap)?

Reply:
Because the rest of the process relies on reading those files to figure out if it needs to build something, decoupling writing those files from the rest of the process seems dangerous.

Yes, we might want to do this when we add a file, like we did for `latest`, but that's only if we don't want to wait for the next release to update the files. I'll think a bit more about this.